pkg/chartutil/save_test.go

```diff
@@ -102,7 +102,7 @@ func TestSave(t *testing.T) {
 				t.Fatal(err)
 			}
 			if c2.Lock == nil {
-				t.Fatal("Expected v2 chart archive to containe a Chart.lock file")
+				t.Fatal("Expected v2 chart archive to contain a Chart.lock file")
```

Comment:
This is a user visible change

Reply:
it's just the output from a failing test, so it isn't a user visible change.